src/layered_config_tree/main.py

```diff
@@ -346,6 +346,18 @@ def to_dict(self) -> NestedDict:
                 result[name] = child.to_dict()  # type: ignore[assignment]
         return result
 
+    def get(self, key: str, default_value: Any = None) -> Any:
+        """Return the LayeredConfigTree or value at the key in the outermost layer of the config tree.
+
+        Parameters
+        ----------
+        key
+            The str we look up in the outermost layer of the config tree.
```

Comment:
Nit: let's not use first-person verbs in our docs.
```suggestion
            The string to look for the outermost layer of the config tree.
```